### PR TITLE
10.0 pos orders history

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by ilmir-k — APPROVED** [review]

File: pos_orders_history/models/pos_orders_history_models.py [hunk 1]
```diff
@@ -13,7 +13,10 @@ class PosConfig(models.Model):
     _inherit = 'pos.config'
 
     orders_history = fields.Boolean("Orders History", help="Show all orders list in POS", default=True)
-    current_day_orders_only = fields.Boolean("Current Day Orders Only", help="Show current day orders only", default=True)
+
+    load_orders_of_last_n_days = fields.Boolean("Orders of last 'n' days", default=True)
```
Comment (yelizariev) [inline pos_orders_history/models/pos_orders_history_models.py]: It should be False by defaul

File: pos_orders_history_return/doc/changelog.rst [hunk 2]
```diff
@@ -3,6 +3,12 @@
 Updates
 =======
 
+`1.0.1`
+-------
+
+- **FIX:** Product price is taken from original order rather than Sale Price
```
Comment (yelizariev) [inline pos_orders_history_return/doc/changelog.rst]: It's not obvious which variant is correct and which is fixed

File: pos_orders_history/models/pos_orders_history_models.py (hunk 1 above)
Comment (yelizariev) [inline pos_orders_history/models/pos_orders_history_models.py]: load_limit_orders or something like that sounds better IMO

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_orders_history/models/pos_orders_history_models.py (hunk 1 above)
Comment (GabbasovDinar) [reply pos_orders_history/models/pos_orders_history_models.py]: @yelizariev Why? We need to load all orders for a specified number of days, not for a specified number of orders

Comment (GabbasovDinar) [reply pos_orders_history/models/pos_orders_history_models.py]: @yelizariev I think first of all that **load_limit_orders** means to load a specified number of orders

**Review by yelizariev — APPROVED** [review]

**Pull request merged**